capabilities.json

```diff
@@ -1284,7 +1284,8 @@
           "type": "object"
         },
         "type": "function"
-      }
+      },
+      "nondeterministic": true
```

Comment:
It would be nice, but we don't have a page yet for this. :thinking: Perhaps a good future work item, and definitely part of the docs work for the "wiring the cache into the decision logger" PR that'll be coming down the road after this PR lands.

Reply:
Good enough in the other PR! Thanks